saleor/product/forms.py

```diff
@@ -20,21 +23,28 @@ def label_from_instance(self, obj):
                 'price': gross(obj.get_price(discounts=self.discounts))}
         return label
 
+    def update_field_data(self, product, cart):
```

Comment:
The product itself is not needed in this function; it could take a `queryset` instead.

Reply:
:+1: